src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/SetAzureRmAnalysisServicesServer.cs

```diff
@@ -26,37 +27,54 @@ namespace Microsoft.Azure.Commands.AnalysisServices
     [Alias("Set-AzureAs")]
     public class SetAzureAnalysisServicesServer : AnalysisServicesCmdletBase
     {
-        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
+        private const string ParamSetDefault = "Defualt";
+        private const string ParamSetDisableBackup = "Disablebackup";
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true, 
+            ParameterSetName = ParamSetDefault,
             HelpMessage = "Name of the server.")]
+        [Parameter(ParameterSetName = ParamSetDisableBackup)]
```

Comment:
@taiwu do we want the `ParamSetDisableBackup` parameter set to include all previous parameters _except_ `BackupBlobContainerUri`? If so, we should add that parameter set to the parameters missing it.

Also, please add the attributes that were available for the previous set to this one (_i.e._, `ValueFromPipelineByPropertyName`, `Position`, `Mandatory`, etc.)